libmamba/src/core/transaction.cpp

```diff
@@ -242,16 +243,13 @@ namespace mamba
         else
         {
             // The specs to install become all the dependencies of the non intstalled specs
-            for_each_to_omit(
-                m_solution.actions,
-                [&](const specs::PackageInfo& pkg)
+            for (const specs::PackageInfo& pkg : m_solution.packages_to_omit())
```

Comment:
I'm not sure if this case is always valid in C++20, I remember some issues with temporary ranges in for-range loop, but I might be mixing up with another case.